### DISCUSSION TITLE
How to enable with version 1.18.x ?

### DISCUSSION THREAD

**Discussion by keepsmilyn** [itzg/docker-minecraft-server#1454] [Q&A]
The latest version of Minecraft is up to 1.18.x now, but whenever I change the "Version" variable to LATEST it doesnt work with this version. It seems to only work with 1.17.1 at most. Is there a newer version to install?

**Comment by itzg** [comment]
That's strange since the image has always queried Mojang's servers for the latest version and 1.18.2 has been resolving fine for lots of other people. Please provide your container config and logs.

**Reply by keepsmilyn** [reply]
Strange as it doesn't seem to be working on any later versions, hence why I asked. 

Maybe its failing the query to the Mojang server, could be a connection of permission issue on our server. :-S

To resolve the issue I ended up installation a different Minecraft Java package for our Synology server. Maybe a complete uninstall and reinstall would solve the problem but I didnt want to lose all the world data. :-S

**Reply by itzg** [reply]
If you could have provided the logs, then I could helped identify more specifics.

**Reply by keepsmilyn** [reply]
Logs and server properties file attached. 

I'm no longer using this container in docker, so can't advise which environment variables were set / tested, but I usually referenced from here: https://minecraft.fandom.com/wiki/Server.properties

Image that was being used: itzg / minecraft-server:latest

[server_properties.txt](https://github.com/itzg/docker-minecraft-server/files/8405882/server_properties.txt)

[logs.zip](https://github.com/itzg/docker-minecraft-server/files/8405880/logs.zip)
